airflow/www_rbac/views.py

```diff
@@ -2003,6 +2003,7 @@ def pool_link(attr):
         pool_id = attr.get('pool')
         if pool_id is not None:
             url = '/taskinstance/list/?_flt_3_pool=' + str(pool_id)
+            pool_id = escape(pool_id)
```

Comment:
@ashb I think this needs to be the following. The `url` would still have non-escaped characters otherwise or am I missing something?

```
if pool_id is not None:
            pool_id = escape(pool_id)
            url = '/taskinstance/list/?_flt_3_pool=' + str(pool_id)
            return Markup("<a href='{url}'>{pool_id}</a>".format(**locals()))

```

Reply:
You're right.